modules/integration/tests-integration/tests-service/src/test/java/org/wso2/carbon/esb/rest/test/header/ContentTypeTestCase.java

```diff
@@ -18,59 +18,85 @@
 
 package org.wso2.carbon.esb.rest.test.header;
 
+import com.sun.net.httpserver.Headers;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.UnknownHostException;
+import org.apache.axiom.om.OMElement;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.DefaultHttpClient;
+import static org.testng.Assert.assertEquals;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 import org.wso2.carbon.integration.common.utils.ClientConnectionUtil;
-import org.wso2.carbon.automation.extensions.servers.httpserver.SimpleHttpClient;
-import org.wso2.carbon.automation.extensions.servers.webserver.SimpleWebServer;
 import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
-
-import java.net.InetAddress;
-import java.net.UnknownHostException;
-import java.util.HashMap;
-import java.util.Map;
-
-import static org.testng.Assert.assertEquals;
+import org.wso2.esb.integration.common.utils.ESBTestCaseUtils;
 
 public class ContentTypeTestCase extends ESBIntegrationTest {
     private Log log = LogFactory.getLog(ContentTypeTestCase.class);
+	private HttpServer server = null;
+	public static String contentType;
```

Comment:
I've mistakenly made it as a static. Thanks a lot for pointing this out and changes are included in PR #368